website/docs/brain/custom-applications-old.mdx

```diff
@@ -0,0 +1,403 @@
+---
+id: brain-apps-old
+title: Developing Custom Applications (OLD)
+---
+
+# Developing Custom Applications
+
+:::caution deprecation warning
+This is out-of-date for brains running `v2.x` Amiga OS software.<br/>
+This tutorial only applies to brains running Amiga OS `v1.x` versions.<br/>
+Please check back for an updated tutorial for brains running `v2.x` Amiga OS software.
+:::
+
+:::tip
+We now have a set of tutorials that walk you through creating custom applications that interact with cameras and drive your Amiga from the brain!
+
+Check them out at [**Brain App Tutorials**](/docs/examples/examples-index#brain-app-tutorials).
+:::
+
+## Quick Tips
+
+* The Amiga user is able to install APT packages.
```

Comment:
These tips are nice but they seem a bit out of context here. When I start reading this, I'm not sure what the `entry.sh` is or what a venv per application means.

Reply:
that's an artifact from the old docs. Removing

Reply:
wait -- this is from the old version that is not supported anymore. We can remove in follow up PR if that's allright @Hackerman342 @mihelich

Reply:
Yeah this page is not rendered if it's not added to sidebars,js. No reason to keep around though because we can always find it in the git history